app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -486,6 +511,14 @@ class ComposeViewModel @Inject constructor(
     private companion object {
         const val TAG = "ComposeViewModel"
     }
+
+    enum class ConfirmationPromptType {
```

Comment:
Nit: For consistency with other code, `ConfirmationKind` would probably be better (see how `...Kind` is used as a suffix fora  lot of enums or sealed classes like this elsewhere in the code).